index.bs

```diff
@@ -2137,6 +2142,54 @@ document loading at which a navigation command will return.
 
 ### Commands ### {#module-browsingContext-commands}
 
+#### The browsingContext.activate Command ####  {#command-browsingContext-activate}
+
+The <dfn export for=commands>browsingContext.activate</dfn> command activates and focuses the given top-level browsing context.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      browsingContext.Activate = {
+        method: "browsingContext.activate",
+        params: browsingContext.ActivateParameters
+      };
+
+      browsingContext.ActivateParameters = {
+        context: browsingContext.BrowsingContext
+      }
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <pre class="cddl">
+     EmptyResult
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.activate">
+
+The [=remote end steps=] with |command parameters| are:
+
+1. Let |context id| be the value of the |command parameters|["<code>context</code>"] field.
+
+1. Let |context| be the result of [=trying=] to [=get a browsing context=] with |context id|.
+
+1. If |context| is not a [=top-level browsing context=], return [=error=] with [=error code=] [=invalid argument=].
+
+1. Run implementation-specific steps so that |context|'s [=traversable navigable=]'s [=system visibility state=] becomes [=visible=]. If this is not possible return [=error=] with error code [=unsupported operation=].
+
+    Note: This can have the side effect of making currently [=visible=] [=navigables=] [=hidden=].
+
+    Issue: It's not clear if we should have explicit requirements for cases like a minimized window; should that cause the window to become unminimized?
```

Comment:
```suggestion
    Note: This can change the underlying OS state by causing the window to become unminimized or by other side effects related to changing the [=system visibility state=].
```

Reply:
@jgraham should we phrase it like this then? (given the comment from Mathias and your response above)

Reply:
That looks fine.